backtrace-library/src/main/java/backtraceio/library/services/BacktraceMetrics.java

```diff
@@ -1,12 +1,15 @@
 package backtraceio.library.services;
 
 import android.content.Context;
+
 import androidx.annotation.NonNull;
 
+import java.security.InvalidParameterException;
```

Comment:
is it used?

Reply:
Yup

Reply:
where? 
<img width="1184" alt="image" src="https://user-images.githubusercontent.com/22329621/207460919-57307930-a3e6-4d8f-94dc-277d898ab1bc.png">